public/index.html

```diff
@@ -1,58 +1,57 @@
 <!DOCTYPE html>
 <html lang="en>">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content ="width=device-width, initial-scale=1.0"/>
-        <title>Discord Soundboard Remote</title>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
-        <link rel="stylesheet" href="styles.css"/>
-        
-    </head>
-    <body id="body">
-        <nav class="nav">
-            <div class="nav-left">
-                <div class="title-container">
-                    <h1>DiscordSoundboardBot</h1>
-                   </div> 
-                   <div class="username-container">
-                            <h2 class="username" id="username"></h2>
-                    </div>
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Discord Soundboard Remote</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body id="body">
+<nav class="nav">
+    <div class="nav-left">
+        <div class="title-container">
+            <h1>DiscordSoundboardBot</h1>
+        </div>
+        <div class="username-container">
+            <h2 class="username" id="username"></h2>
+        </div>
+    </div>
+    <div id="avatar-container" class="avatar-container">
+        <img id="avatar" src="" class="avatar" alt="" width="50px">
+        <div id="log-out-menu" class="log-out-menu log-out-menu-hide">
+            <img id="log-out-pointer" class="log-out-pointer" src="log-out-pointer.png" alt="">
+            <button id="log-out-button" class="btn log-out-button" onclick="document.location.href = '/logout'">log out</button>
```

Comment:
I'd rather have this be an `a` with a `href` property, but I couldn't be arsed to style it to look the same as it does now as a button.